### PR TITLE
Improve text shown for past events when none exist 

### LINKED ISSUE #361: [Bug]: Past events uses incorrect empty state of upcoming events
## Description
View past events for a partner with none:

image

### PR DESCRIPTION
**Pull request by r-ferrier**
Fixes #361 

## Description

- When a partner has > 20 events and `past events` is selected, if none exist then the text shown now is 

> There were no past events.

- A test has been added to catch this edge case

Please let me know what the copy should be here if this is not suitable - I just went with something simple for now!

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -185,6 +185,9 @@ t key =
         EventsEmptyText ->
             "There are no upcoming events on this date. Check back for updates!"
 
+        PreviousEventsEmptyTextAll ->
+            "There were no past events."
```
Comment (katjam) [inline src/Copy/Text.elm:189]:
```suggestion
            "There have been no events in the recent past"
```

**Review by katjam — APPROVED** [review]
This looks great. I've added a suggested change to the text, but not sure if it makes it better or worse. Thought I'd add in case it prompted you to think of something better.

I don't remember how far back we go when importing events but I think even if there were never any events with a Partner, it's nicer for public to think there may have been.

Approving because if you think my change confuses things, I'm happy for it to be merged as is.

File: src/Copy/Text.elm (hunk 1 above)
Comment (r-ferrier) [reply src/Copy/Text.elm:189]: This is much clearer, I will change!

**Review by katjam — APPROVED** [review]
Let's go with this.

**Comment by r-ferrier** [comment]
@katjam Unfortunately the build is now failing! Have tried re-running it to no avail. What's the best thing for me to do here - is this a regular occurrence? Should I make an empty commit to trigger a rebuild? Or is this something I'd need to check out in cloudflare once I have access?

**Comment by katjam** [comment]
I will look at it. I think it's to do with Cloudflare and the way it manages node dependencies.

**Comment by katjam** [comment]
I think maybe with this one, the failure was too many running at the same time. I'm going to trigger again.
